Sources/PackageDescription/Package.swift

```diff
@@ -147,3 +147,15 @@ private func dumpPackageAtExit(_ package: Package, fileNo: Int32) {
     dumpInfo = (package, fileNo)
     atexit(dump)
 }
+
+// MARK: URL Normalization
+
+extension String {
+    private func normalizedURL() -> String {
+        var url = self.lowercased()
+        if !url.hasSuffix(".git") {
```

Comment:
It's added to the `libc` target which is imported in the file, so I believe, yes.